core/modules/layout/css/grid-float.css

```diff
@@ -950,3 +960,63 @@
     margin-left: 91.666667%;
   }
 }
+
+/**
+ * Override flexbox values in case they're set
+ */
+.col-xs-first,
+.col-xs-last,
+.col-sm-first,
+.col-sm-last,
+.col-md-first,
+.col-md-last,
+.col-lg-first,
+.col-lg-last,
+.col-xl-first,
+.col-xl-last {
+  -webkit-box-ordinal-group: unset;
+  -webkit-order: unset;
+  -ms-flex-order:unset;
+  order: unset;
+}
+
+.row-xs-top,
+.row-xs-center,
+.row-xs-bottom,
+.row-sm-top,
+.row-sm-center,
+.row-sm-bottom,
+.row-md-top,
+.row-md-center,
+.row-md-bottom,
+.row-lg-top,
+.row-lg-center,
+.row-lg-bottom,
+.row-xl-top,
+.row-xl-center,
+.row-xl-bottom {
+  -webkit-box-align: unset;
+  -webkit-align-items: unset;
+  -ms-flex-align: unset;
```

Comment:
The MS documentation doesn't say it supports either `unset` or `none` for this property, though I'm not sure it matters. The default value is `stretch` according to https://msdn.microsoft.com/en-us/library/hh673531%28v=vs.85%29.aspx